fish_speech/train.py

```diff
@@ -13,48 +11,39 @@
 from lightning.pytorch.strategies import DDPStrategy
 from omegaconf import DictConfig, OmegaConf
 
-os.environ.pop("SLURM_NTASKS", None)
-os.environ.pop("SLURM_JOB_NAME", None)
-os.environ.pop("SLURM_NTASKS_PER_NODE", None)
+# Set environment variables
+os.environ["USE_LIBUV"] = "0"
+os.environ["SLURM_NTASKS"] = os.environ.get("SLURM_NTASKS", None)
```

Comment:
we can remove line 16-18

Reply:
yes  if we don’t need to handle SLURM-related environment variables.

Reply:
We don't need that, it should work fine in most slurm envs.